training/src/main/java/wolox/training/models/BookDTO.java

```diff
@@ -0,0 +1,110 @@
+package wolox.training.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.node.IntNode;
+import com.google.common.base.Joiner;
+import java.io.IOException;
+import java.time.LocalDate;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import wolox.training.exceptions.CouldNotCreateBookFromDTO;
+
+public class BookDTO {
+
+    private String ISBN;
+    private String title;
+    private String subtitle;
+    private List<String> publishers;
+    private String publishDate;
+    private Integer numberOfPages;
+    private List<String> authors;
+
+    private String image;
+
+    public String getISBN() {
+        return ISBN;
+    }
+
+    public void setISBN(String ISBN) {
+        this.ISBN = ISBN;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public String getSubtitle() {
+        return subtitle;
+    }
+
+    public void setSubtitle(String subtitle) {
+        this.subtitle = subtitle;
+    }
+
+    public List<String> getPublishers() {
+        return publishers;
+    }
+
+    public void setPublishers(List<String> publishers) {
+        this.publishers = publishers;
+    }
+
+    public String getPublishDate() {
+        return publishDate;
+    }
+
+    public void setPublishDate(String publishDate) {
+        this.publishDate = publishDate;
+    }
+
+    public Integer getNumberOfPages() {
+        return numberOfPages;
+    }
+
+    public void setNumberOfPages(Integer numberOfPages) {
+        this.numberOfPages = numberOfPages;
+    }
+
+    public List<String> getAuthors() {
+        return authors;
+    }
+
+    public void setAuthors(List<String> authors) {
+        this.authors = authors;
+    }
+
+    public String getImage() { return image; }
```

Comment:
![image](https://user-images.githubusercontent.com/12162035/58508868-7ec78700-816b-11e9-9c6d-23c20c771896.png)
I just made a pull and that's how I see it (image uploaded). The authors getter is also "compressed" in one line. I didn't do it, it's somehow automatically done. I changed it now for it to be in different lines, but do you know what should I do to prevent it from compressing again?

Reply:
Note that most methods are collapsed, but once you click on them you can expand them (this is something IntelliJ does in order to help you visualize methods and save space, it's called Code Folding and you can configure it in Settings > Editor > General > Code Folding). However, the methods getImage() and setImage() are not compressed, just missing a line jump 😅(you can verify it by opening the file in a plain text editor).

Reply:
Thanks!! I would have never noticed that :smile: